test/Microsoft.AspNetCore.Razor.Language.Test/TagHelperBinderSyntaxTreePassTest.cs

```diff
@@ -14,6 +14,132 @@ namespace Microsoft.AspNetCore.Razor.Language
     public class TagHelperBinderSyntaxTreePassTest
     {
         [Fact]
+        public void Execute_CanHandleSingleLengthAddTagHelperDirective()
```

Comment:
Eh, don't want the tests to be tied directly to the implementation. 3 different features, they just happen to all go through the same path today.